src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs

```diff
@@ -115,7 +115,7 @@ private static void ProcessEndRequest(IExecutionContext executionContext, Activi
             var httpResponse = responseContext.HttpResponse;
             if (httpResponse != null)
             {
-                int statusCode = (int)httpResponse.StatusCode;
+                var statusCode = (int)httpResponse.StatusCode;
```

Comment:
Seems I applied  some offered by VS auto-refactoring. Although some changes are reasonable, but since we would like to keep changes related only to renaming of package it was reverted.